service/capability/audio_player_agent.cc

```diff
@@ -611,12 +611,17 @@ void AudioPlayerAgent::onSyncContext(const std::string& ps_id, std::pair<std::st
         aplayer_listener->renderDisplay(render_info.first, render_info.second);
 }
 
-void AudioPlayerAgent::onReleaseContext(const std::string& ps_id)
+bool AudioPlayerAgent::onReleaseContext(const std::string& ps_id, bool unconditionally)
 {
     nugu_dbg("AudioPlayer release context");
 
+    unsigned int no_clear_count = 0;
+
     for (auto aplayer_listener : aplayer_listeners)
-        aplayer_listener->clearDisplay();
+        if (!aplayer_listener->clearDisplay(unconditionally))
+            no_clear_count++;
+
```

Comment:
App이 꼭 지워야 하는 가이드를 따르지 않는 경우를 위해 아래 로그가 추가되면 추후 도움이 될 것 같습니다.

```suggestion
if (unconditionally && no_clear_count)
    nugu_warn("should clear display if unconditionally is true!!");
```

Reply:
추가했습니다.